### DISCUSSION TITLE
in Windows Subsystem for Linux (WSL), I am having an error when i compile my script .

### DISCUSSION THREAD

**Discussion by Benterkiii** [Cyfrin/foundry-full-course-cu#1073] [Q&A]
here is my terminal:
...
benterkiii@DESKTOP-BC7F9AS:~/foundry-f23/foundry-found-me-f23$ forge script script/DeploySimpleStorage.s.sol
[⠢] Compiling...
[⠔] Compiling 16 files with 0.8.22
[⠒] Solc 0.8.22 finished in 247.59ms
Error: 
Compiler run failed:
Error (2904): Declaration "script" not found in "lib/forge-std/src/Script.sol" (referenced as "forge-std/Script.sol").
 --> script/DeploySimpleStorage.s.sol:5:1:
  |
5 | import {script} from "forge-std/Script.sol";
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ...

**Comment by Naman1729** [comment]
Hey @Benterki-Zinou , change the spelling of `script` to `Script` in import.
```solidity
import {Script} from "forge-std/Script.sol";
```
